pgo/cmd/update.go

```diff
@@ -104,23 +107,31 @@ var UpdateClusterCmd = &cobra.Command{
 	Long: `Update a PostgreSQL cluster. For example:
 
     pgo update cluster mycluster --autofail=false
-    pgo update cluster mycluster myothercluster --autofail=false
-    pgo update cluster --selector=name=mycluster --autofail=false
-    pgo update cluster --all --autofail=true`,
+    pgo update cluster mycluster myothercluster --disable-autofail
+    pgo update cluster --selector=name=mycluster --disable-autofail
+    pgo update cluster --all --enable-autofail`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if Namespace == "" {
 			Namespace = PGONamespace
 		}
 
 		if len(args) == 0 && Selector == "" && !AllFlag {
 			fmt.Println("Error: A cluster name(s) or selector or --all is required for this command.")
-		} else {
-			if util.AskForConfirmation(NoPrompt, "") {
-				updateCluster(args, Namespace)
-			} else {
-				fmt.Println("Aborting...")
-			}
+			os.Exit(1)
+		}
+
+		// if both --enable-autofail and --disable-autofail are true, then abort
+		if (EnableAutofailFlag && DisableAutofailFlag) || !(EnableAutofailFlag || DisableAutofailFlag) {
+			fmt.Println("Error: Cannot set --enable-autofail and --disable-autofail simultaneously")
+			os.Exit(1)
+		}
```

Comment:
Does this mean that at least one of these flags is required? Seems like if that were the case, then wouldn't it be better to have a single flag that defaults to the opposite value? In my view, I'm not sure I'm just not sure it makes sense to have two separate flags to control the same thing. :thinking:

Reply:
Let me check, there is a chance I screwed that up.

Reply:
Yup, I did screw that one up. Good catch.